Wireframe/index.html

```diff
@@ -1,33 +1,48 @@
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <header>
-      <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </header>
-    <main>
-      <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
-        <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
-        </p>
-        <a href="">Read more</a>
-      </article>
-    </main>
-    <footer>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </footer>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Wireframe</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <h1>Wireframe</h1>
+  <p class="subtitle">Learn about README files, wireframes, and Git branches</p>
+
+  <div class="container">
+    <div class="card">
+      <img src="https://via.placeholder.com/800x300" alt="Placeholder image">
+      <h2>What is the purpose of a README file?</h2>
+      <p>The primary purpose of a <code>README.md</code> file is to serve as a user guide and introduction to a project,
+      particularly in software development and GitHub repositories. It provides essential information such as the
+      project's purpose, setup instructions, usage guidelines, and how others can contribute. It is often the
+      first document a user or developer will read when exploring a new project or codebase.</p>
+      <a class="button" href="#">Read more</a>
+    </div>
+
+    <div class="row">
+      <div class="card">
+        <img src="https://via.placeholder.com/400x200" alt="Placeholder image">
+        <h2>What is the purpose of a wireframe?</h2>
+        <p>A wireframe is a basic, skeletal representation of a website or application's layout and structure.
+        It focuses on the arrangement of content and features without detailed design elements like colors or images.
+        Wireframes act as blueprints, helping designers, developers, and stakeholders understand the user interface
+        before proceeding to the visual designphase.</p>
+        <a class="button" href="#">Read more</a>
+      </div>
+
+      <div class="card">
+        <img src="https://via.placeholder.com/400x200" alt="Placeholder image">
+        <h2>What is a branch in Git?</h2>
+        <p>In Git, a branch represents an independent line of development. It acts as an abstraction for the edit/stage/commit
+        process, allowing you to work on different features, bug fixes, or experiments in isolation. Branching enables
+        teams to collaborate more effectively and manage multiple development streams within the same repository.</p>
+        <a class="button" href="#">Read more</a>
+      </div>
+    </div>
```

Comment:
Footer is not positioned fixed. While transforming your index.html please make sure you also make your footer fixed as well